build.proj

```diff
@@ -33,6 +33,8 @@
     <ExeExtension>.exe</ExeExtension>
     <ExeExtension Condition=" '$(OS)' != 'Windows_NT' "></ExeExtension>
     <DotnetStage0>$(Stage0Path)/dotnet$(ExeExtension)</DotnetStage0>
+
+    <DotnetCliBuildDirectory>$(MSBuildThisFileDirectory)/build_projects/dotnet-cli-build/</DotnetCliBuildDirectory>
```

Comment:
Fixed. SPecifically, I replaced all but one instances of MSBuildThisFileDirectory with RepoRoot and rearranged property definitions to allow RepoRoot to be the root path.